apps/web/src/services/asset-inventory/collector/collector-history/CollectorHistoryPage.vue

```diff
@@ -54,13 +54,13 @@ const fields: DataTableField[] = [
     { label: 'Created', name: 'created_at' },
     { label: 'Duration', name: 'duration', sortable: false },
 ];
-const statusList = [
+const statusList = computed(() => [
     { name: JOB_SELECTED_STATUS.ALL, label: i18n.t('INVENTORY.COLLECTOR.HISTORY.ALL') },
     { name: JOB_SELECTED_STATUS.PROGRESS, label: i18n.t('INVENTORY.COLLECTOR.HISTORY.IN_PROGRESS') },
     { name: JOB_SELECTED_STATUS.SUCCESS, label: i18n.t('INVENTORY.COLLECTOR.HISTORY.SUCCESS') },
     { name: JOB_SELECTED_STATUS.FAILURE, label: i18n.t('INVENTORY.COLLECTOR.HISTORY.FAILURE') },
     { name: JOB_SELECTED_STATUS.CANCELED, label: i18n.t('INVENTORY.COLLECTOR.HISTORY.CANCELED') },
-];
+]);
```

Comment:
Suggestion: I think you don't need this code!

Reply:
How about applying computed to state.statusList below code?

Reply:
i removed it!